backend/src/index.ts

```diff
@@ -43,6 +49,11 @@ io.on("connection", (socket) => {
   registerRoomEvents(io, socket);
 });
 
+// Serve the index.html file for all routes
+app.get("*", (req, res) => {
+  res.sendFile(path.join(__dirname, "../../frontend/dist/index.html"));
+});
```

Comment:
If you go to the backend URL, it will host the files that `frontend` creates from the react code